content-cache/src/state.py

```diff
@@ -181,6 +191,8 @@ def from_integration_data(cls, data: ops.RelationDataContent) -> "LocationConfig
         backends_str = data.get(BACKENDS_FIELD_NAME, "").strip()
         fail_timeout = data.get(FAIL_TIMEOUT_FIELD_NAME, "").strip()
         backends_path = data.get(BACKENDS_PATH_FIELD_NAME, "").strip()
+        healthcheck_path = data.get(HEALTHCHECK_PATH_FIELD_NAME, "").strip()
+        healthcheck_interval = int(data.get(HEALTHCHECK_INTERVAL_FIELD_NAME, "2000").strip())
```

Comment:
It is a bit unclear why 2000 is chosen as the default.
Is there a special reason for it?

Reply:
I just reused the value from the example provided in the documentation from the module. https://github.com/openresty/lua-resty-upstream-healthcheck
This is something we can change for sure. Or at least put it as a constant maybe?

Reply:
I think we should use a good default value for the charm.
The current default of 2000 works out to 2 seconds, which seems a bit too frequent to me.
What is the default value used in the package? Maybe set it to that. 

If the value needs to be updated then remember to update the config file as well: 
`content-cache-backends-config/config.yaml`

Reply:
The module itself has no default.
I was looking at HA-proxy, and it has a similar default of 2s: https://www.haproxy.com/documentation/haproxy-configuration-tutorials/service-reliability/health-checks/#change-the-interval
```
Change the interval 
By default, the load balancer sends a health check every two seconds. Change this by adding the inter argument to the server line. In the next example, we send a health check every four seconds:

```

As the goal of active healthchecks is to limit impact on end-users, it makes sense to have a small interval. For a static healthcheck endpoint, 1req/2s is not generating any load.

Reply:
Ok, that seems to be a good value then.
Thanks for the responses.